target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -11,6 +11,13 @@ board_config_update
 board=$(ipq806x_board_name)
 
 case "$board" in
+ad7200)
+	ucidef_set_led_usbdev "usb1" "USB 1" "${board}:blue:usb_1" "2-1"
+	ucidef_set_led_usbdev "usb2" "USB 2" "${board}:blue:usb_3" "4-1"
+	ucidef_set_led_netdev "wan" "WAN" "${board}:blue:wan" "eth0"
+	ucidef_set_led_netdev "lan" "LAN" "${board}:blue:lan" "br-lan"
```

Comment:
Reagarding this line a fix was commited a few minutes ago: https://github.com/lede-project/source/commit/5aace5a5fbcf4a80c6765b605ffdd86d2cfadfb0#diff-9b6a658a3fcc5295de51e97afb3dd5a3L18.

I guess the same applies to this line as well. Would you please change it here as well and test if it still works.